### ISSUE TITLE
vl-source-wms - incorrect extParms parsing

### ISSUE THREAD

**Issue by pingwiniasty** [ghettovoice/vuelayers#52]
Hi! 
I'm trying to use Polish WMS server with vl-source-wms.
But extParams was incorrect parsed, look at my sample config:
```
{
    name: 'wms',
    title: 'Geoportal',
    visible: true,
    url: 'http://mapy.geoportal.gov.pl/wss/service/img/guest/ORTO/MapServer/WMSServer',
    layers: 'ORTOFOTOMAPA',
    styles: 'default',
    extParams: {CSR: 'EPSG:4320'}
}
```
But it's generate incorrect url with additional param **0=CSR**
`http://mapy.geoportal.gov.pl/wss/service/img/guest/ORTO/MapServer/WMSServer?0=CSR&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image/png&TRANSPARENT=true&LAYERS=ORTOFOTOMAPA&STYLES=default&WIDTH=256&HEIGHT=256&CRS=EPSG:3857&BBOX=2118222.927838806,6511211.817444455,2123114.897649057,6516103.787254706`

**Comment by ghettovoice** [comment]
Hi @pingwiniasty ,
you are right, there is an issue with helper that prepares `extParams`. I'll fix it.
But for projection definition `vl-source-wms` has `projection` property (it work like in native OpenLayers https://openlayers.org/en/latest/apidoc/ol.source.TileWMS.html). Also if you want to use custom projection (only `EPSG:4326` and `EPSG:3857` predefined by default) you should register it in [proj4js](https://github.com/proj4js/proj4js) regestry:
```js
// main.js
import proj4 from 'proj4'

proj4.defs('EPSG:4320', '%projection definition in Proj4 format%') // https://epsg.io/
// now OpenLayers and VueLayers can find that projection by code EPSG:4326
```
```html
...
<vl-source-wms projection="EPSG:4320"></vl-source-wms>
...
```

Thanks!

**Comment by ghettovoice** [comment]
I just release the new patched version v0.10.8. It should resolve issue with invalid `extParams` handling.
Thank you!

**Comment by become-iron** [comment]
There is a redundant [code line](https://github.com/ghettovoice/vuelayers/commit/e311482f55385d26347714fe287cd739ba4f4c5e#diff-7e93e536316433c0c64248cec8ae5471R74) with `console.log(s)`

**Comment by ghettovoice** [comment]
fixed v0.10.9

**Comment by pingwiniasty** [comment]
You are so fast! Thanks!
